services/status.go

```diff
@@ -2,15 +2,11 @@ package services
 
 import (
 	"encoding/json"
-	"eth2-exporter/db"
-	"eth2-exporter/version"
-	"os"
-	"time"
 )
 
 // Report the status of a particular service, will add current Pid and executable name
 func ReportStatus(name, status string, metadata *json.RawMessage) {
-	pid := os.Getpid()
+	/* pid := os.Getpid()
```

Comment:
This change does not seem right. Why have you removed the function's body?

Reply:
Temporary workaround until a implementation for BIDS-1329 gets merged into this branch, won't change